Preprocessing/OneHotEncoder.ecl

```diff
@@ -0,0 +1,83 @@
+/*##############################################################################
+## HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems.  All rights reserved.
+############################################################################## */
+
+IMPORT $.^ as ML_Core;
+IMPORT $.Types;
+NumericField := ML_Core.Types.NumericField;
+l_cFeatures := Types.OneHotEncoder.cFeatures;
+
+/**
+  * OneHotEncoder
+  * Support Myraid interface.
```

Comment:
Should say:  "Supports Myriad Interface."  Otherwise it is not clear if you are saying that it needs to support Myriad, or if it does.